packages/jsii-pacmak/test/expected.jsii-calc/python/src/jsii_calc/__init__.py

```diff
@@ -1033,114 +1054,183 @@ def prop(self, value: str):
 
     class Derived(Base, metaclass=jsii.JSIIMeta, jsii_type="jsii-calc.DerivedClassHasNoProperties.Derived"):
         """
-        Stability:
-            experimental
+        stability
+        :stability: experimental
         """
         def __init__(self) -> None:
             jsii.create(DerivedClassHasNoProperties.Derived, self, [])
 
 
 
-@jsii.data_type_optionals(jsii_struct_bases=[scope.jsii_calc_lib.MyFirstStruct])
-class _DerivedStruct(scope.jsii_calc_lib.MyFirstStruct, jsii.compat.TypedDict, total=False):
-    anotherOptional: typing.Mapping[str,scope.jsii_calc_lib.Value]
-    """This is optional.
+@jsii.data_type(jsii_type="jsii-calc.DerivedStruct", jsii_struct_bases=[scope.jsii_calc_lib.MyFirstStruct], name_mapping={'anumber': 'anumber', 'astring': 'astring', 'first_optional': 'firstOptional', 'another_required': 'anotherRequired', 'bool': 'bool', 'non_primitive': 'nonPrimitive', 'another_optional': 'anotherOptional', 'optional_any': 'optionalAny', 'optional_array': 'optionalArray'})
+class DerivedStruct:
```

Comment:
Don't we want this to inherit from `MyFirstStruct`? Did we test struct inheritence?

Reply:
Not even sure what that means.

Reply:
Okay it will be used for nominal typing, probably.